stdlib/list.mli

```diff
@@ -73,6 +73,14 @@ val nth_opt: 'a list -> int -> 'a option
 val rev : 'a list -> 'a list
 (** List reversal. *)
 
+val init : int -> (int -> 'a) -> 'a list
+(** [List.init n f] returns a fresh list of length [n],
+   with element number [i] initialized to the result of [f i].
+   In other terms, [List.init n f] tabulates the results of [f]
+   applied to the integers [0] to [n-1].
+
+   Raise [Invalid_argument] if [n < 0].*)
```

Comment:
A workaround would be to define both `init` and `init_rev`. But is it really interesting?